### ISSUE TITLE
ValueError: 'ci' filter requires columns 'ci_lo', 'ci_hi'

### ISSUE THREAD

**Issue by holtgrewe** [etal/cnvkit#143]
What do I need to do to get the ci_lo and ci_hi columns for `cnvkit.py call --filter ci` to work?

```
Calling copy number with thresholds: -1.1 => 0, -0.25 => 1, 0.2 => 2, 0.7 => 3
Applying filter 'cn'
Filtered by 'cn' from 3237 to 1552 rows
Applying filter 'ampdel'
Filtered by 'ampdel' from 1552 to 54 rows
Applying filter 'ci'
Traceback (most recent call last):
  File "/fast/users/mholtgr/venv/bin/cnvkit.py", line 13, in <module>
    args.func(args)
  File "/fast/users/mholtgr/venv/lib/python3.4/site-packages/cnvlib/commands.py", line 791, in _cmd_call
    args.thresholds)
  File "/fast/users/mholtgr/venv/lib/python3.4/site-packages/cnvlib/commands.py", line 846, in do_call
    outarr = getattr(segfilters, filt)(outarr)
  File "/fast/users/mholtgr/venv/lib/python3.4/site-packages/cnvlib/segfilters.py", line 29, in wrapped_f
    raise ValueError(msg.format(filtname, *colnames))
ValueError: 'ci' filter requires columns 'ci_lo', 'ci_hi'
```

**Comment by holtgrewe** [comment]
I'm seeing a similar issue for 'sem'

**Comment by etal** [comment]
Sorry, this is a new feature. You need to first run the .cns file through `segmetrics` with the options to calculate and add those columns:

    cnvkit.py segmetrics Sample.cns  --ci --sem -o Sample.segmetrics.cns
    cnvkit.py call Sample.segmetrics.cns --filter ci --filter sem

**Comment by etal** [comment]
Additional caveats:

- In CNVkit 0.8.0, the `ci` filter would only work when used alone, not alongside other filters. Fixed in CNVkit 0.8.1.
- `ci` and `sem` are still mutually exclusive. I'm not sure about the math for updating the SEM values where two or more segments are merged (via the initial CI pass).
